installer/packer/scripts/harbor/harbor.cfg

```diff
@@ -0,0 +1,113 @@
+## Configuration file of Harbor
+
+#The IP address or hostname to access admin UI and registry service.
+#DO NOT use localhost or 127.0.0.1, because Harbor needs to be accessed by external clients.
+hostname = reg.mydomain.com
+
+#The protocol for accessing the UI and token/notification service, by default it is http.
```

Comment:
This will come up in the product security review - should be secure by default and with appropriate minimum TLS versions/approved cipher suites.